distribution/changelog.txt

```diff
@@ -6,7 +6,8 @@
 - Feature: [#22172] [Plugin] Expose ride satisfaction ratings to the plugin API.
 - Feature: [#22184] [Plugin] Expose staff statistics to the plugin API.
 - Feature: [#22213] [Plugin] Allow plugins to focus on textboxes in custom windows.
-- Feature: [#22272] [Plugin] Expose ride vehicle’s current track type via car trackLocation.
+- Feature: [#22272] [Plugin] Expose ride vehicle’s current track type to the plugin API.
```

Comment:
This changelog item is not from this PR.

Reply:
correct. I copied my previous PR number by mistake.